fastlane/docs/PluginsTroubleshooting.md

```diff
@@ -50,12 +50,12 @@ Your `Pluginfile` should look something like this
 ```ruby
 # Autogenerated by fastlane
 
-gem 'fastlane-plugin-xcversion'
+gem 'fastlane-plugin-ruby'
```

Comment:
@ohwutup pointed out to me that fastlane has a built-in action called xcversion. Seeing that, it wasn't clear to me how the xcversion example plugin works at all. This was just being a little defensive, just in case